homeassistant/components/verisure/alarm_control_panel.py

```diff
@@ -55,6 +55,16 @@ def unique_id(self) -> str:
         """Return the unique ID for this alarm control panel."""
         return self.coordinator.entry.data[CONF_GIID]
 
+    @property
+    def device_info(self) -> dict[str, Any]:
+        """Return device information about this entity."""
+        return {
+            "name": "Verisure Alarm",
+            "manufacturer": "Verisure",
+            "model": "VBox",
```

Comment:
Is this the human readable name ?

Reply:
Yes, VBox (their product name for the Verisure Box) is the main hub everything connects to (and thus "the alarm system").